action/receipt.go

```diff
@@ -12,24 +12,67 @@ import (
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/golang/protobuf/proto"
 
+	"github.com/iotexproject/go-pkgs/byteutil"
 	"github.com/iotexproject/go-pkgs/hash"
+	"github.com/iotexproject/iotex-address/address"
 	"github.com/iotexproject/iotex-proto/golang/iotextypes"
 
 	"github.com/iotexproject/iotex-core/pkg/log"
 )
 
+// constants
+const (
+	StakingProtocolID   = "staking"
+	WithdrawAmountTopic = "withdrawAmount"
```

Comment:
shall we move this as a var after InContractTransfer?

Reply:
we could, and is there a reason we want to use var instead of const? just want to learn
i always prefer to use const whenever possible.

Reply:
As we design SystemLog, the topic, like "InContractTransfer", is more like an enum, which starts with 0x0...0. It is not saying this is a better way, but it is better to be consistent.